definitions/ext-istio_service/definition.yml

```diff
@@ -0,0 +1,47 @@
+domain: EXT
+type: ISTIO_SERVICE
+synthesis:
+  rules:
+  - identifier: "label.service.istio.io/canonical-name"
```

Comment:
How long can this field be? we probably want to set encoding to true

Reply:
@srvaroa I don't know the maximum limit, so to be safe, I've added `encodeIdentifierInGUID: true` to each of the rules

Reply:
Thanks!